hack/helm/install-unittest-plugin.sh

```diff
@@ -33,7 +33,7 @@ fi
 
 helm plugin uninstall unittest || true
 curl \
-  -L "https://github.com/helm-unittest/helm-unittest/releases/download/v${desired_version}/helm-unittest-${PLATFORM}-${ARCH}-${desired_version}.tgz" \
+  -L "https://github.com/helm-unittest/helm-unittest/releases/download/${desired_version}/helm-unittest-${PLATFORM}-${ARCH}-${desired_version//v}.tgz" \
```

Comment:
This magic trick removes `v` -> you can read about it here https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
nice! :+1: